opal.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'sourcemap', '~> 0.1.0'
   spec.add_dependency 'hike', '~> 1.2'
   spec.add_dependency 'ast', '>= 2.3.0'
-  spec.add_dependency 'parser', '= 2.4.0.2'
+  spec.add_dependency 'parser', '= 2.5.0.4'
```

Comment:
rubocop should be added as development dependency with the exact version we're using, also the version in codeclimate should be updated https://github.com/iliabylich/opal/blob/c97062039d3b54cc1385e4e70d658591e3e5dd26/.codeclimate.yml#L6